src/Microsoft.SqlTools.Hosting/Utility/CommandOptions.cs

```diff
@@ -64,13 +65,13 @@ public CommandOptions(string[] args)
             {
                 if (!string.IsNullOrEmpty(ErrorMessage) || ShouldExit)
                 {
-                    Console.WriteLine(Usage);
+                    Console.WriteLine((string) Usage);
```

Comment:
I'd use the `.ToString()` method instead of casting it into a string

Reply:
I'll remove the cast since it's unnecessary.